rpc/src/test/java/com/navercorp/pinpoint/rpc/util/ListUtilsTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2019 NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.navercorp.pinpoint.rpc.util;
+
+import java.util.ArrayList;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ListUtilsTest {
+
+    @Test
+    public void testAddIfValueNotNull() {
+        ArrayList<String> strings = new ArrayList<String>();
+
+        Assert.assertTrue(ListUtils.addIfValueNotNull(strings, "foo"));
+        Assert.assertTrue(strings.get(0).equals("foo"));
+
+        Assert.assertFalse(ListUtils.addIfValueNotNull(strings, null));
+    }
+
+    @Test
+    public void testAddAllIfAllValuesNotNull() {
+        ArrayList<String> strings = new ArrayList<String>();
+
+        Assert.assertTrue(
+                ListUtils.addAllIfAllValuesNotNull(
+                        strings, new String[]{"a", "b", "c"}));
+        Assert.assertTrue(strings.get(0).equals("a"));
+        Assert.assertTrue(strings.get(1).equals("b"));
```

Comment:
Add check for "c" as here are only 3 elements?

Reply:
Done